src/main/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapper.java

```diff
@@ -12,24 +12,24 @@
     public static final String BUILD_STATUS = "build";
     public static final String DOWN_STATUS = "down";
     public static final String ERROR_STATUS = "error";
-    private static final String CREATING_STATUS = "creating";
-    private static final String AVAILABLE_STATUS = "available";
-    private static final String ATTACHING_STATUS = "attaching";
-    private static final String DETACHING_STATUS = "detaching";
-    private static final String IN_USE_STATUS = "in-use";
-    private static final String MAINTENANCE_STATUS = "maintenance";
-    private static final String DELETING_STATUS = "deleting";
-    private static final String AWAITING_TRANSFER_STATUS = "awaiting-transfer";
-    private static final String ERROR_DELETING_STATUS = "error_deleting";
-    private static final String BACKING_UP_STATUS = "backing-up";
-    private static final String RESTORING_BACKUP_STATUS = "restoring-backup";
-    private static final String ERROR_BACKING_UP_STATUS = "error_backing-up";
-    private static final String ERROR_RESTORING_STATUS = "error_restoring";
-    private static final String ERROR_EXTENDING_STATUS = "error_extending";
-    private static final String DOWNLOADING_STATUS = "downloading";
-    private static final String UPLOADING_STATUS = "uploading";
-    private static final String RETYPING_STATUS = "retyping";
-    private static final String EXTENDING_STATUS = "extending";
+    public static final String CREATING_STATUS = "creating";
```

Comment:
@llvieira I'm not sure we need this to be public. If you are using only for tests, shouldn't be protected?

Reply:
I made it protected now.